source/docs/installation/pro-installation.rst

```diff
@@ -43,12 +43,22 @@ Online
 
 .. tab-set::
 
-   .. tab-item:: FRC
+   .. tab-item:: FRC (Pro Only)
+
+      .. important:: This vendordep is for robot projects that are **only** using Pro (licensed) devices.
 
       Paste the following URL in `WPILib VS Code <https://docs.wpilib.org/en/stable/docs/software/vscode-overview/3rd-party-libraries.html#libraries>`__ :guilabel:`Install New Libraries (Online)`
 
       - ``https://maven.ctr-electronics.com/release/com/ctre/phoenixpro/PhoenixPro-frc2023-latest.json``
 
+   .. tab-item:: FRC (Pro & Phoenix 5)
+
+      .. important:: This vendordep is for robot projects that are using **both** Pro & Phoenix 5 devices (licensed and non-licensed).
+
+      Paste the following URL in `WPILib VS Code <https://docs.wpilib.org/en/stable/docs/software/vscode-overview/3rd-party-libraries.html#libraries>`__ :guilabel:`Install New Libraries (Online)`
+
+      - ``https://maven.ctr-electronics.com/release/com/ctre/phoenixpro/PhoenixProAnd5-frc2023-latest.json``
```

Comment:
Add the important block below here from the `Phoenix 5 & Phoenix Pro Compatibility` section